installer/chart/volcano/templates/admission.yaml

```diff
@@ -98,3 +108,24 @@ spec:
   selector:
     app: volcano-admission
   sessionAffinity: None
+
+---
+apiVersion: batch/v1
```

Comment:
how about init container for admission controller?

Reply:
The difference is we need to run the init-container every time when new admission pod is created, but job would only execute once.

Reply:
That should be ok, as the script will remote the old one. If using job, how to make sure secret is generated before admission pod running?

Reply:
Pod will wait until the secret is ready because it will mount the secret. when used in init-container it will delete&recreate the secret every time when pod recreated.

Reply:
that's ok to me. BTW, Do we delete secret when uninstall volcano?

Reply:
yeah, we need investigate all of the uncleaned resource and remove them when uninstalling.